src/metabase/driver/sql/query_processor.clj

```diff
@@ -621,19 +621,24 @@
 
 (defn rewrite-fields-to-force-using-column-aliases
   "Rewrite `:field` clauses to force them to use the column alias regardless of where they appear."
-  [form]
-  (mbql.u/replace form
-    [:field id-or-name opts]
-    [:field id-or-name (-> opts
-                           (assoc ::add/source-alias        (::add/desired-alias opts)
-                                  ::add/source-table        ::add/none
-                                  ;; sort of a HACK but this key will tell the SQL QP not to apply casting here either.
-                                  ::nest-query/outer-select true
-                                  ;; used to indicate that this is a forced alias
-                                  ::forced-alias            true)
-                           ;; don't want to do temporal bucketing or binning inside the order by or breakout either.
-                           ;; That happens inside the `SELECT`
-                           (dissoc :temporal-unit :binning))]))
+  ([form]
+   (rewrite-fields-to-force-using-column-aliases form {:is-breakout true}))
```

Comment:
Having `true` as the default looks suspicious. Where is the call with `:is-breakout false`? Also, I would expect here parameters that result in the behavior we had before the change (something like `nil`).

Reply:
younre right, this is a typo, let me undo this and check